Automattic/jetpack
# Connection: add spinner for the "Connect" button

#### Changes proposed in this Pull Request:
- copy the Calypso spinner as an `@automattic/jetpack-components` component
- add the said spinner to the connection screen.

#### Jetpack product discussion
p1630326033081800-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Disconnect Jetpack.
2. Go to the connection screen, click "Set up Jetpack".
3. Confirm that upon clicking, the text in the button gets replaced with the spinner.
4. Disconnect Jetpack, go to Jetpack Backup connection page, confirm that the spinner is there too.

**The spinner:**
<img width="309" alt="Screen Shot 2021-08-30 at 2 59 05 PM" src="https://user-images.githubusercontent.com/1341249/131390632-331caefc-bd9d-4faa-a291-df672eba1300.png">
